A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/EICAS/ECAM/A320_Neo_UpperECAM.html

```diff
@@ -20,6 +20,7 @@
 
 <script type="text/html" import-script="/JS/dataStorage.js"></script>
 <script type="text/html" import-script="/Pages/A32NX_Utils/NXDataStore.js"></script>
+<script type="text/html" import-script="/Pages/A32NX_Core/A32NX_FWC.js"></script>
```

Comment:
I'm not super happy about importing thise here. An alternative would be to move the logic nodes to A32NX_Utils, but I wasn't sure about the convention and how it would be used.

Reply:
The logic nodes seem to be incredibly versatile. I would suggest moving them to the Utils.

Reply:
Sounds good. Any suggestion for a file name? I see the previous ones are in some kind of NX namespace - or is that somethnig else? How about `NXLogic.js`?

Reply:
Sounds good to me.

Reply:
Done.